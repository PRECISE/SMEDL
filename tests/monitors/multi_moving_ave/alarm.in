{
    "fmt_version": [2, 0],
    "channel": "in",
    "event": "measurement",
    "params": [0, 11.0],
    "aux": {
        "arbitrary": "json data"
     }
}
