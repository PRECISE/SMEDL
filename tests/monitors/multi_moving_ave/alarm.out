{
    "fmt_version": [2, 0],
    "channel": "_Frontend_raiseAlarm",
    "event": "Frontend.raiseAlarm",
    "identities": [],
    "params": [],
    "aux": {
        "arbitrary": "json data"
     }
}
